.github/workflows/main.yml

```diff
@@ -16,19 +16,53 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This workflow contains a single job called "build"
+  geotests:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        version: ["3.9", "3.10", "3.11"]
```

Comment:
This lib mentions is supports python 3.8+, so I think it would be nice to add `3.8` as well

Reply:
Done fe828df901f1d6bad884b2a93338b910611ca3ed